semantic_model_generator/validate/schema.py

```diff
@@ -0,0 +1,76 @@
+from typing import Dict
```

Comment:
can you add a readme to this directory, or a top level comment here, that explains what this file is doing and how to modify/update if needed